include/pose_calculations.hpp

```diff
@@ -28,9 +28,11 @@ typedef struct pose_der_s {
 
 enum CoordinateSystem { UNCHANGED=0, NED, ENU};
 enum UpAxis { NOTDETECTED=-1, X=0, Y, Z };
-enum ArenaDirection{RIGHT=0, FAR_SIDE, LEFT, NEAR_SIDE};
+enum ArenaDirection{RIGHT=0, FAR_SIDE, LEFT, NEAR_SIDE, TRUE_NORTH};
 
 pose_t transform_pose(const CoordinateSystem co,
+                      const ArenaDirection co_north,
+                      const double true_north_deg,
```

Comment:
Is there a need for this to be a double? We've adhered to floating precision throughout most of the rest of this project so maybe we should also make this a float.

Reply:
it's weird that we did that in the first place, but no, not required. I'll make it float